stringUpdateScript.js

```diff
@@ -0,0 +1,33 @@
+fs = require('fs');
```

Comment:
I like the idea of the script.
Couple of questions:
1. Could we do some auto-detection on the files?
2. Could we process file by file to prevent having that much stuff in memory?

Reply:
1. I added the script to package.json so every time `npm run package` is called, the strings in the `sample` folder will be updated. The `sample` folder contains all the files that we want to upload to Zanata, since it contains the most up to date strings. I'm not sure how to go about adding auto-detection, but I think adding the script to package.json should update it frequently enough.
2. The script runs through metadata files, probably only a few kB each, which are created by the gulpfile, which is run when `npm run package` is called. You could process it file by file, but I think checking all the files at once is more consistent.